interface/themes/style_light.css

```diff
@@ -1259,3 +1259,16 @@ body.admin-layout input[type="button"] {
 .error-border{
   border: 1px solid red !important;
 }
+
+.tinylogocontainer
+{
+  display: inline-block;
+  border: none;
+}
+
+.tinylogopng
+{
+  display: inline-block;
+  width: 44px;
+  height: 22px;
```

Comment:
You should add a sample logo1 and logo2 file. Just do what you did in logo_3.png below. Make logo1 show logo1 in the logo and logo2 show logo2 in the logo. And make them the typical size. Then users can learn how to use them by simply turning them on and replacing them with a similar sized image.